Mathlib/Data/Dfinsupp/Order.lean

```diff
@@ -17,13 +17,19 @@ This file lifts order structures on the `α i` to `Π₀ i, α i`.
 
 ## Main declarations
 
-* `dfinsupp.order_embedding_to_fun`: The order embedding from finitely supported dependent functions
+* `Dfinsupp.orderEmbeddingToFun`: The order embedding from finitely supported dependent functions
   to functions.
 
 -/
 
+-- porting note: The unusedVariables linter has false positives in code like
+--    variable (α) [∀ i, Zero (α i)]
+-- marking the  `i` as unused. So disable that lint for now.
+-- Likely issue https://github.com/leanprover/lean4/issues/2088, so remove when that is fixed
+set_option linter.unusedVariables false
```

Comment:
Please use the syntax
```
set_option linter.unusedVariables false in
theorem/def fooBar (α) [∀ i, Zero (α i)] ....
```
to disable the linter locally.

Reply:
Applying this to `variables` lines seems to change their semantics, and the later code turn red. 

I would not waste time on this, not linting for unused variables on the file doesn’t hurt, and we’ll remove it once the bug is fixed anways.

Reply:
Apparently, the linter is happy with:
```
variable (α)
variable [∀ i, Zero (α i)]
```

So I did that, then deleted `set_option`.